kotest-property/src/jvmMain/kotlin/io/kotest/property/seed/seedio.kt

```diff
@@ -22,9 +22,15 @@ actual fun readSeed(path: TestPath): Long? {
 fun seedDirectory(): Path = Paths.get(System.getProperty("user.home")).resolve(".kotest").resolve("seeds")
 
 fun seedPath(path: TestPath): Path {
-   return seedDirectory().resolve(path.value)
+   return seedDirectory().resolve(escape(path.value))
 }
```

Comment:
If `seedPath` ever throws an exception, then whole execution will crash, because even if writing-failed seeds is disabled, reading them is always enabled.

To help prevent this, can `seedPath(...)` be nullable?

```suggestion
fun seedPath(path: TestPath): Path? = runCatching {
   seedDirectory().resolve(escape(path.value))
}.getOrNull()
```

If not, can at least its usage inside `readSeed(...)` be inside the try/catch?

Reply:
Yep good, idea. It won't let me merge this as conflicts, can you redo ?

Reply:
```suggestion
fun seedPath(path: TestPath): Path? = runCatching {
   seedDirectory().resolve(escape(path.value))
}.getOrNull()
```

I don't think GitHub handles suggestions on deleted lines

Reply:
I'll apply it manually.